### PR TITLE
pcl: Update to PCL 1.8.0.

### PR DESCRIPTION
**Pull request by mikepurvis**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [x] Removed the `revision` line, if any, when bumping the version number?
- [x] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

Closes #3512.

### CODE REVIEW

File: pcl.rb [hunk 1]
```diff
@@ -31,36 +31,10 @@ def message
...
-      sha256 "6eec64c55c282a0e81c462da92bb07d5e945080ddc1ac3a9d2039dc402caa4be"
-    end
-  end
-
-  bottle do
-    sha256 "14342057a9125ae005f7b54a5863c3043ce68accd5967cee6f520707d8862bd2" => :el_capitan
-    sha256 "a7b98799063432d87031cb40df82c8bd9797d64fa14becc0f047b678f955ffa3" => :yosemite
-    sha256 "8f116eafa5db19551f4ac6a56e525879fc2efc8fe0b3788585f9367b2ce40786" => :mavericks
+    url "https://github.com/PointCloudLibrary/pcl/archive/pcl-1.8.0.tar.gz"
+    sha256 "9e54b0c1b59a67a386b9b0f4acb2d764272ff9a0377b825c4ed5eedf46ebfcf4"
```
Comment (dpo) [inline pcl.rb]: > Updates to existing formula: have you
- [X] Not altered the bottle section?

Please don't alter the bottle section ;-)

Comment (mikepurvis) [inline pcl.rb]: Oh, I misunderstood the guideline. So I leave it alone with the out of date bottles and let the brew bot swap them out?

**Comment by mikepurvis** [comment]
My local build is now giving "Error: Empty installation", but `/usr/local/Cellar/pcl/1.8.0` is full of stuff, and after a `brew link pcl`, it seems to be fully working. So I'm not sure what's going on with that. We'll see what the CI build comes back with.

**Comment by dpo** [comment]
Looks all good. Thank you!

**Pull request closed without merging**